packages/commitlint-config/package.json

```diff
@@ -15,7 +15,8 @@
     "npm-package-json-lint": "7.1.0",
     "prettier": "3.1.1",
     "tsup": "8.0.1",
-    "typescript": "5.3.3"
+    "typescript": "^5.3.3",
```

Comment:
Zero installs sound good to me. If only because it encodes a working state of the application into git, regardless of package managers working or being accessible.